### ISSUE TITLE
Node.js Collection Blogs from April 3 and April 10

### ISSUE THREAD

**Issue by ZibbyKeaton** [nodejs/evangelism#286]
Hey All - I would love for folks to review and provide feedback by EOD Wednesday (April 5) on the following blog: https://docs.google.com/document/d/1cjroc_Z_pIzvUJgh8VnNPhTwhzbkLSS6jgvccci1OzE/edit#. I had a few edits, but I think there are a few more items that need to be clarified in the "Typescript is a friend." Those that have the tech knowledge will understand it better than me - I feel a bit lost here.

Additionally, I would like to add this to the Medium publication: https://medium.com/@nodejs/meet-the-speaker-committee-for-node-js-interactive-8e62e1641713 It details our speaker committee for Node.js Interactive this year. Please let me know if this group is okay with me posting or would rather I didn't by EOD Wednesday (April 5). I encourage this group to submit and/or share with those that they think would have awesome presentations. First time conference speakers are welcome!

I'm planning to share a few tweets to encourage folks to submit a blog for Node.js Collection. If you see any posts on Medium that you think would be perfect for the collection, please share here. The more education and information we can provide the community, the better :)

**Comment by vdeturckheim** [comment]
I really like the second article. Can we publish it ASAP since the CFP deadline is approaching?

**Comment by ZibbyKeaton** [comment]
If folks are open to this, we can do that. Can we accelerate this to giving me a yes or no by EOD?

**Comment by davidfekke** [comment]
I don't have access to the first article, but TypeScript is a great subject
for a post. I like the second speaker committee intro.

**Comment by ZibbyKeaton** [comment]
@davidfekke looks like you have access now. If you could just check out that blog and that section in particular that would be awesome.

**Comment by anaptfox** [comment]
@ZibbyKeaton I can help review the Typescript article as well. I just requested access. 

The speaker article is great! It's nice to see the people behind the mask.

**Comment by davidfekke** [comment]
Zibby,

I added my comments. I asked for some additional information around the
TypeScript tooling they used.

Thanks,
David

**Comment by ZibbyKeaton** [comment]
@anaptfox thank you. Just request to be added to the google document and provide your comments.

**Comment by ZibbyKeaton** [comment]
The second iteration of defining streams is in. You can find it here: https://medium.com/@jessica.quynh.tran/a-brief-history-of-node-streams-pt-2-bcb6b1fd7468. Please let me know if anyone has comments/changes that they want to make to this by EOD Friday. If not, then I'll plan to add to Node.js Collection on Monday.

**Comment by ZibbyKeaton** [comment]
And this blog has been added: https://medium.com/the-node-js-collection/lessons-learned-one-year-with-node-js-and-the-enterprise-49f1adabf14a.

**Comment by ZibbyKeaton** [comment]
Hey All - I received the following question from someone on our Medium page:

"I would like to know how to develop NodeJS applications using typescript. There are some references there, but I don't see any formal introduction. Any advice?"

Does anyone have a good place to lead this person or is this something that we think would make for a good blog post? And if so, any recommendations on authors?

**Comment by davidfekke** [comment]
The gentleman who can be seen speaking at the JaxNode user group in my Medium post on "How to start a Node.js User Group" was actually doing a presentation on that subject. His name is Brady Merkel, and he is a V.P. at Bank of America here in Jacksonville. I will reach out to him and see if he publish his slides and source code from that presentation.

**Comment by ZibbyKeaton** [comment]
@davidfekke that would be awesome. And if he finds the time and wants to convert his slides into a blog post then "Yah!"

**Comment by joshgav** [comment]
@ZibbyKeaton 

> Does anyone have a good place to lead this person [on TypeScript and Node.js]?

One of our local TypeScript experts recommended this: http://brianflove.com/2016/03/29/typescript-express-node-js/

cc @bowdenk7 @danielrosenwasser

**Comment by bpmerkel** [comment]
@davidfekke Thank you for remembering my presentation! You might be the only one! LOL I had to go back and review my notes to find just how little content I had on the subject... Nonetheless, I read Brian Love's 3 blog posts and think his content is a much better walkthrough of the (maturing) TypeScript + node.js story. He even takes it further with Mongoose, Mocha, and Chai... somewhere I haven't gone yet. My experience was from the Visual Studio (and .NET) perspective, and I did have some challenges I never resolved. Namely, I had a “collision” of TypeScript targets from within the same project (I wanted to target ES5+CommonJS for client-side scripts; and ES6 for server-side. I never did solve the split to two projects to enable this.) The other issue was conflicting Promises (supported natively with node.js—but required an additional library for client side.) Recommendation? Don't wait for me... send 'em Brian's way... and perhaps commission Brian to update his posts with more specific case studies using latest versions of tooling. Convergence is happening to ease the pain of past complexity, and accelerate change for the better.
